packages/common/editors/__tests__/CheckboxEditor.spec.js

```diff
@@ -1,27 +1,45 @@
 const React          = require('react');
 const ReactDOM = require('react-dom');
 const TestUtils      = require('react-dom/test-utils');
+const { mount } = require('enzyme')
```

Comment:
Small personal preference change: I typically prefer to require third party packages first, and then local packages.